app/src/main/java/org/simple/clinic/facility/Facility.kt

```diff
@@ -41,6 +43,9 @@ data class Facility(
     // have group UUID until they're synced again.
     val groupUuid: UUID?,
 
+    @Embedded(prefix = "location_")
+    val location: Coordinates?,
```

Comment:
The only reason this is nullable is because of backward-compatibility, correct?

Reply:
Two reasons:
1. Backward compatibility
2. We don't have the location of every facility yet